css/style.css

```diff
@@ -0,0 +1,558 @@
+body {
+  font-family: 'Chelsea Market', cursive;
+  background-color: rgb(222, 238, 222);
+}
+
+img {
+  filter: grayscale(90%);
+}
+
+.main-container {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+}
+
+.nav-bar {
+  display: flex;
+  border-bottom: 1px solid black;
+  justify-content: space-around;
+  padding: 15px 0 15px 0;
+  background-color: lightgray;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+}
+
+.nav-mid {
+  display: flex;
+  width: 650px;
+  justify-content: space-between;
+}
+
+.nav-mid div,
+.nev-left div,
+.nav-right div {
+  border: 1px solid black;
+  border-radius: 2px;
+  background-color: rgb(102, 102, 102);
+  color: white;
+  font-weight: bold;
+  padding: 10px;
+}
+
+.nav-mid div:hover {
+  cursor: pointer;
+}
+
+.nav-left {
+  padding-left: 20px;
+}
+
+.nav-left img {
+  width: 30px;
+  background: black;
+}
+
+.search-article {
+  width: 320px;
+}
+
+.search-article input {
+  width: inherit;
+  border-radius: 8px;
+  height: 30px;
+}
+
+.dont-miss {
+  display: flex;
+  justify-self: center;
+  font-size: 34px;
+  box-shadow: black;
+  max-width: fit-content;
+  margin-top: 100px;
+  grid-column: 3/6;
+  box-sizing: border-box;
+  background-color: grey;
+  color: honeydew;
+  margin-left: 140px;
+}
+
+.dont-miss h1 {
+  text-align: center;
+}
+
+.articles {
+  display: grid;
+  grid-column: 1/-1;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  margin-top: 20px;
+  padding: 20px;
+}
+
+.left-article {
+  grid-column: 1/2;
+  grid-row: 1/2;
+  background-color: rgb(116, 110, 110);
+  color: white;
+}
+
+.right-article {
+  grid-column: 2/3;
+  grid-row: 1/2;
+  background-color: gray;
+  color: white;
+}
+
+.left-article1 {
+  grid-column: 1/2;
+  grid-row: 2/3;
+  background-color: rgb(121, 99, 99);
+  color: white;
+}
+
+.right-article1 {
+  grid-column: 2/3;
+  grid-row: 2/3;
+  background-color: rgb(153, 138, 138);
+  color: white;
+}
+
+.img {
+  width: 85px;
+  height: 85px;
+  background-color: rgb(59, 59, 59);
+  border-radius: 4px;
+  transform: rotate(-20deg);
+  margin-top: 20px;
+  box-shadow: 0 10px rgba(46, 41, 41, 0.178);
+}
+
+.imgTitle:hover .img {
+  transform: rotate(360deg);
+}
```

Comment:
- [OPTIONAL] First of all, kudos on making the rotation! I will have advice, so this animation will look better and smooth.

Add these lines to `.img`:
```
transform-origin: 0 100%;
transition: all 0.5s ease;
```

The changes mentioned before will help you modify the desired axis.

Also, change `.imgTitle:hover .img` to the following:
```
.img:hover {
  transform: rotate(0);
}
```

Now the animation will only trigger when you hover over the image.

Reply:
:100: